contrib/ProtobufLogger.py

```diff
@@ -114,6 +114,7 @@ def printResponse(self, message):
                 if rr.HasField('class'):
                     rrclass = getattr(rr, 'class')
                 rrtype = rr.type
+                rrudr = rr.udr
```

Comment:
Doesn't this generate an error if the RR does not have a `udr` field set, which might happen since it's optional?

Reply:
Good point